config.cmake.template

```diff
@@ -50,13 +50,48 @@
 # set(BDDCML_ROOT "$ENV{HOME}/local/BDDCML/bddcml-2.2")
 
 ### Python setting ###############################################################
-# USE_PYTHON - only use python if this is set to "yes"
+# USE_PYTHON - use embedded python if this is set to "yes"
 # We add python library only on explicit request since it leads to tons of errors when debugging with valgrind.
 #
 # set(USE_PYTHON "yes")
 
 # PYTHON_COPY - make copy of python files, necessary for release builds
 # set(PYTHON_COPY "yes")
+#  - Following variables MUST ALSO BE SET:
+#    - PYTHON_COPY_ROOT
+#           - absolute path to python root e.g.:
+#             set(PYTHON_COPY_ROOT "/usr/lib/python2.7")
+#           
+#    - PYTHON_COPY_PACKAGES 
+#           - list of all core modules which will be copied
+#             each entry represents folder name realtive to root, e.g.:
+#             set(PYTHON_COPY_MODULES encodings json)
+#           - by default we want almost all basic packages to python can work 
+#             correctly, example:
+#             set(PYTHON_COPY_PACKAGES
+#                 bsddb compiler ctypes curses distutils email encodings hotshot importlib 
+#                 json lib-dynload lib-tk lib2to3 logging multiprocessing 
+#                 plat-x86_64-linux-gnu pydoc_data sqlite3 test unittest wsgiref xml)
+#             
+#   - PYTHON_COPY_DIST_PACKAGES
+#           - list of all 3rd party packages (installed using pip or other tool)
+#             each entry is either
+#                - relative path:
```

Comment:
It depends on platform. Some tools install packages to `dist-packages` but some to `site-packages`. So we cannot detect it reliably.